aiida_quantumespresso/calculations/projwfc.py

```diff
@@ -25,7 +27,17 @@ class ProjwfcCalculation(NamelistsCalculation):
         ('PROJWFC', 'plotboxes', False),
     ]
     _default_parser = 'quantumespresso.projwfc'
-    _internal_retrieve_list = [NamelistsCalculation._PREFIX + '.pdos*']
+
+    xml_filename = 'data-file-schema.xml'
+
+    _internal_retrieve_list = [
+        NamelistsCalculation._PREFIX + '.pdos*',
+    ]
+    _retrieve_temporary_list = [
+        os.path.join(
+            NamelistsCalculation._default_parent_output_folder, f'{NamelistsCalculation._PREFIX}.save', xml_filename
+        )
+    ]
```

Comment:
Any particular reason to put have this in the temporary retrieved? Is the full data parsed and stored in output nodes?